test/float16.jl

```diff
@@ -14,6 +16,25 @@ g = Float16(1.)
 @test isless(g, f)
 @test !isless(f, g)
 
+@test convert(Bool,Float16(0.)) == false
+@test convert(Bool,Float16(0.1)) == true
```

Comment:
I would fix, but I have two different copies of Julia, both compiling at the same time and no clean working copies...